.eslintrc.js

```diff
@@ -1260,6 +1262,16 @@ module.exports = {
         ...require('eslint-config-prettier/@typescript-eslint').rules,
       },
     },
+    /**
+     * Enterprise Search Prettier override
+     * Lints unnecessary backticks - @see https://github.com/prettier/eslint-config-prettier/blob/main/README.md#forbid-unnecessary-backticks
+     */
```

Comment:
@spalger Let me know if the comments + location look okay to you! Happy to tweak copy, newlines, etc., as needed to help make things clearer to future devs